sources/packages/backend/libs/sims-db/src/entities/notification-metadata.type.ts

```diff
@@ -5,4 +5,5 @@ export interface NotificationMetadata {
   disbursementId?: number;
   applicationNumber?: string;
   assessmentId?: number;
+  scholasticStandingId?: number;
```

Comment:
This can be removed based on the outcome of https://github.com/bcgov/SIMS/pull/5541/changes#r2632689288.